tests/test_request_helper.py

```diff
@@ -19,14 +20,20 @@ def test_response_ok_200_return_true(self):
     def test_response_ok_400_return_false(self):
         response = Response()
         response.status_code = 400
-        result = request_helper.response_ok(response)
-        self.assertEqual(result, False)
+        with self.assertRaises(RetryableError):
+            request_helper.response_ok(response)
```

Comment:
I would of expected that a 400 client error would not be retry-able whereas a 500 server error would be